feature-libs/order/jest.ts

```diff
@@ -0,0 +1,24 @@
+// uncomment when we switch the whole lib to jest
+/*
+Object.defineProperty(window, 'CSS', { value: null });
+Object.defineProperty(window, 'getComputedStyle', {
+  value: () => {
+    return {
+      display: 'none',
+      appearance: ['-webkit-appearance'],
+    };
+  },
+});
+
+Object.defineProperty(document, 'doctype', {
+  value: '<!DOCTYPE html>',
+});
+Object.defineProperty(document.body.style, 'transform', {
+  value: () => {
+    return {
+      enumerable: true,
+      configurable: true,
+    };
+  },
+});
+*/
```

Comment:
Do we need this dead config file?

Reply:
It is a required file. I copied it from other libs.